src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -436,7 +437,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
    */
   private[this] def processAccountsForBlocks(
       updates: List[BlockTagged[List[AccountId]]],
-      votingData: List[(BlockHash, List[Voting.BakerRolls])]
+      votingData: Map[BlockHash, List[Voting.BakerRolls]]
```

Comment:
The idea to use a map here is to avoid a linear search. Now we should have a simple lookup by key, but I can't see it applied in the following code

Reply:
ok, I removed it from here